### PR TITLE
chore: sec lake updates

### PR DESCRIPTION
**Pull request by jcountsNR**
Added notes for SQS queue and Subscriber creation

### CODE REVIEW

File: src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-security-lake-monitoring-integration.mdx [hunk 1]
```diff
@@ -7,37 +7,108 @@ tags:
...
+      <td>
+      S3  
+      </td>
+    </tr>
+    <tr>
+      <td>
+      `Account ID`  
+      </td>
+      <td>
+      Insert the AWS `account ID` for the account that you want to set up monitoring for
```
Comment (anshrma) [inline src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-security-lake-monitoring-integration.mdx:58]: Thats not correct. 
This is customer's AWS account ID where you plan on installing the New Relic provided serverless application. SL grants IAM permissions to this account

File: src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-security-lake-monitoring-integration.mdx [hunk 2]
```diff
@@ -7,37 +7,108 @@ tags:
...
+      </td>
+    </tr>
+    <tr>
+      <td>
+      `External ID`
+      </td>
+      <td>
+      Insert your New Relic `AccountID`
```
Comment (anshrma) [inline src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-security-lake-monitoring-integration.mdx:66]: Link them to [External ID](https://docs.aws.amazon.com/security-lake/latest/userguide/subscriber-data-access.html#subscriber-external-id).

File: src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-security-lake-monitoring-integration.mdx [hunk 3]
```diff
@@ -7,37 +7,108 @@ tags:
...
+      <td>
+      SQS queue  
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+5. Select **Create**.
+
+Your `SecurityLakeSQSArn` and `SecurityLakeSubscriberRoleArn` are generated here. Take note of them because you'll need them for the next step.
```
Comment (anshrma) [inline src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-security-lake-monitoring-integration.mdx:82]: There are no fields like this in AWS UI. You need to explain how to correlate output in UI with what is expected in function paramters

File: src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-security-lake-monitoring-integration.mdx [hunk 4]
```diff
@@ -7,37 +7,108 @@ tags:
...
-Install our dedicated [Amazon Security Lake log forwarder](/docs/logs/forward-logs/aws-lambda-sending-security-logs-s3/) but instead of Lambda trigger, use AWS SQS service. Set up variables accordingly.
-
-## Logs
-
-To find your logs on New Relic, go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Logs** and set **Attributes** to `product.name`, choose one or few of your interest.
-
-The followings source types of logs are currently supported:
+To enable this integration, set up an S3 log forwarder. We suggest using our application for ease and convenience, but you can also set your own.
+
+## Configuration steps [#steps]
```
Comment (anshrma) [inline src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-security-lake-monitoring-integration.mdx:18]: * This is missing guidance on [a] How would a customer configure logs to be ouput to begin with ?

```
Complete the steps mentioned in Amazon Security Lake [Getting Started Guide]( https://docs.aws.amazon.com/security-lake/latest/userguide/getting-started.html).
```

* Pre-Requisite

```
Complete the pre-requisites mentioned under `data access` subscriber type as mentioned at [Security Lake Subscriber Guide](https://docs.aws.amazon.com/security-lake/latest/userguide/subscriber-data-access.html#prereqs-creating-subscriber)
```

* Guidance on regions.. What is NR recommendation to install this integration per region basis or use rollup region

```
Consolidate data from multiple regions into a single rollup. We recommend consolidating multiple regions so you can avoid repeating the steps in the next section for each region. 

Amazon Security Lake collects security logs and events across AWS Regions for enabled services. AWS stores data in different Amazon S3 buckets per each region. For more details, see [managing multiple regions](https://docs.aws.amazon.com/security-lake/latest/userguide/manage-regions.html).
```

Comment (jcountsNR) [reply src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-security-lake-monitoring-integration.mdx:18]: Ok, let me review this and update it, I'll make another commit shortly.

**Review by anshrma — DISMISSED** [review]

File: src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-security-lake-monitoring-integration.mdx [hunk 5]
```diff
@@ -4,40 +4,126 @@ tags:
...
+  For more details, see [managing multiple regions](https://docs.aws.amazon.com/security-lake/latest/userguide/manage-regions.html).
+</Callout>
 
-The followings source types of logs are currently supported:
+## Prerequisites [#prerequisites]
+
+Before enabling this integration, first make sure these steps are completed for Security Lake:
+
+1. Complete the steps mentioned in the Amazon Security Lake [Getting started guide]( https://docs.aws.amazon.com/security-lake/latest/userguide/getting-started.html).
+2. Complete the prerequisites mentioned under `data access` subscriber type as mentioned in [Security Lake subscriber guide](https://docs.aws.amazon.com/security-lake/latest/userguide/subscriber-data-access.html#prereqs-creating-subscriber)
```
Comment (jcountsNR) [inline src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-security-lake-monitoring-integration.mdx:28]:
```suggestion
2. Complete the prerequisites to setting up a subscriber with data access by following the steps in the [Security Lake subscriber guide](https://docs.aws.amazon.com/security-lake/latest/userguide/subscriber-data-access.html#prereqs-creating-subscriber).
```

**Pull request merged**